### DISCUSSION TITLE
ESP-IDF Arduino Component (IDFGH-13883)

### DISCUSSION THREAD

**Discussion by Hiren017** [espressif/arduino-esp32#10464] [Q&A] [answered]
Till which version of ESP-IDF, there is a support for Arduino Component?

**Comment by Jason2866** [comment, marked as answer]
For Core 2.0.17 use IDF 4.4.x. Core 3.0.5 needs IDF 5.1.4 and upcoming core 3.1.0 is based on IDF 5.3.1

**Reply by Hiren017** [reply]
thanks @Jason2866 . May I know how long will it take for the upcoming core 3.1.0 to be released? or when will I be able to use IDF v5.2.2?

**Reply by Jason2866** [reply]
IDF 5.2.x is not planned to be used as base for any Arduino core. So never.
There is already the 3.1.0rc1 In my experience already very stable. Idk when the release version will be out. I guess it will be this year.

**Reply by Hiren017** [reply]
ok @Jason2866 . Thanks for making it clear.
